### PR TITLE
docs(tasks): deprecates task-member endpoints, describes owners, perm…

### PR DESCRIPTION
**Pull request by jstirnaman**
…issions:

- deprecates task-owner, task-member endpoints given the roles have no effect. (Closes #483, #412, #413, #414, #417, #416, #415)
- moves UnauthorizedRequestError to common.
- misc. grammar fixes.
- clarify auths, link to signin.
- formatting fixes.

### CODE REVIEW

**Comment by jstirnaman** [comment]
@sunbryely-influxdata I haven't rebased this on your outstanding PRs yet.

File: src/common/paths/orgs.yml [hunk 1]
```diff
@@ -3,7 +3,20 @@ get:
   tags:
     - Organizations
     - Security and access endpoints
-  summary: List all organizations
+  summary: List organizations
+  description: |
+    Retrieves a list of [organizations]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#organization/).
+
+    To limit which tasks are returned, pass query parameters in your request.
+    If no query parameters are passed, InfluxDB returns all tasks up to the default `limit`.
```
Comment (sunbryely-influxdata) [inline src/common/paths/orgs.yml:11]:
```suggestion
    To limit which organizations are returned, pass query parameters in your request.
    If no query parameters are passed, InfluxDB returns all organizations up to the default `limit`.
```

File: src/common/paths/orgs.yml [hunk 2]
```diff
@@ -13,54 +26,76 @@ get:
...
+        An organization ID.
+        Only returns the organization with this ID.
     - in: query
       name: userID
       schema:
         type: string
-      description: Filter organizations to a specific user ID.
+      description: |
+        A user ID.
+        Only returns organizations that have this user as a member? Owner?
```
Comment (sunbryely-influxdata) [inline src/common/paths/orgs.yml:45]: Should be returning organizations where the user is a member or an owner

**Review by sunbryely-influxdata — APPROVED** [review]

File: src/common/paths/orgs.yml (hunk 2 above)
Comment (jstirnaman) [inline src/common/paths/orgs.yml:45]:
```suggestion
        Only returns organizations where this user is a member or owner.
```

**Pull request merged**